packages/core/src/html/includePanelProcessor.ts

```diff
@@ -165,6 +165,15 @@ function _deleteIncludeAttributes(node: MbNode) {
   delete node.attribs.omitFrontmatter;
 }
 
+function buildGetNextFootnodeNumber() {
+  let footnoteNumber = 1;
```

Comment:
Can we just let this start from 0 in this case? So that we don't have this confusing add and then delete below